package-lock.json

```diff
@@ -62,12 +63,47 @@
 				"lint-staged": "~13.0.3",
 				"msw": "~0.47.3",
 				"msw-storybook-addon": "~1.6.3",
+				"plop": "~3.1.1",
 				"prettier": "~2.7.1",
 				"source-map-explorer": "~2.5.3",
 				"tsconfig-paths-webpack-plugin": "~4.0.0",
 				"typescript": "~4.8.4"
 			}
 		},
+		"../js-generators/packages/plop-next-folder-structure": {
+			"version": "1.0.0",
+			"extraneous": true,
+			"license": "MIT",
+			"devDependencies": {
+				"plop": "3.1.1"
+			},
+			"peerDependencies": {
+				"plop": ">=3.1.1"
+			}
+		},
+		"../js-react-nuts-and-bolts/packages/plop-next-folder-structure": {
+			"version": "1.0.0",
+			"extraneous": true,
+			"license": "MIT",
+			"devDependencies": {
+				"plop": "3.1.1"
+			},
+			"peerDependencies": {
+				"plop": ">=3.1.1"
+			}
+		},
+		"../react-nuts-and-bolts/packages/plop-next-ts-generators": {
```

Comment:
What are these, relative paths? How did this finish here?

Reply:
Aaaa, it's a malware xD

Reply:
Now you are reviewing lock file xD I see